library/Cube/Cube.php

```diff
@@ -4,11 +4,19 @@
 
 namespace Icinga\Module\Cube;
 
+use Icinga\Application\Modules\Module;
 use Icinga\Exception\IcingaException;
+use Icinga\Module\Cube\ProvidedHook\Icingadb\IcingadbSupport;
 use Icinga\Web\View;
 
 abstract class Cube
 {
+    /** @var string Prefix for slice params */
+    public const SLICE_PREFIX = 'slice.';
+
+    /** @var bool Whether the icingadb backend is in use */
+    public const IS_USING_ICINGADB = null;
```

Comment:
Can you set it to `false` as default? This would not require adjusting `IdoCube.php`.

Reply:
Changed it [recently](https://github.com/Icinga/icingaweb2-module-cube/pull/99#discussion_r1048133763). The default to `false` causes `Cube::isUsingIcingaDb()` to never check the second condition. View scripts call this method and need the second condition.

Reply:
I see.